articles/quickstart/native/ios-swift/04-calling-apis.md

```diff
@@ -15,46 +15,27 @@ budicon: 546
   ]
 }) %>
 
-The reason for implementing authentication, in the first place, is to protect information. In this case, your information is a resource served from a server of any sort. Auth0 provides a squad of tools to assist you with end-to-end authentication in an application. We recommend that you conform to RFC standards by sending valid authentication tokens through an authorization header.
+Auth0 provides a set of tools for protecting your resources with end-to-end authentication in your application. 
```

Comment:
Should this link to another guide for protecting your API?

Reply:
Could you advise further on this? The article below describes some of the steps you can do to call an API and then provides a link to the API documentation(/api/authentication). Is there other guide I should be linking to in line 18?

Reply:
It might be better to provide this link at the start vs at the end?

Reply:
I agree. I moved the "For more information" note right below the intro.